<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetplayDialog</class>
 <widget class="QDialog" name="NetplayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Netplay game - melonDS</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnLeaveGame">
       <property name="text">
        <string>Leave game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStartGame">
       <property name="text">
        <string>Start game</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblInputBufferSize">
       <property name="text">
        <string>Input Buffer size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spnInputBufferSize">
       <property name="minimum">
         <number>0</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="tvPlayerList"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
